Helm/opensearch-dashboards/values.yaml

```
@@ -59,22 +58,32 @@ securityContext:
   runAsNonRoot: true
   runAsUser: 1000
 
-
 config:
   dashboards.yml:
     server:
-        name: dashboards
-        host: 0.0.0.0
-
-    ## Dashboards TLS Config
-        ssl:
-          enabled: false
-          key: /usr/share/dashboards/certs/dashboards-key.pem
-          certificate: /usr/share/dashboards/certs/dashboards-crt.pem
+      name: dashboards
+      host: 0.0.0.0
+
+      ## Dashboards TLS Config
+      ssl:
+        enabled: false
+        key: /usr/share/dashboards/certs/dashboards-key.pem
+        certificate: /usr/share/dashboards/certs/dashboards-crt.pem
     opensearch.ssl.certificateAuthorities: /usr/share/dashboards/certs/dashboards-root-ca.pem
 
 priorityClassName: ""
 
+probes:
+  readiness:
+    enabled: true
+    custom:
+      tcpSocket:
+        port: http
+      periodSeconds: 15
+  liveness:
+    enabled: true
+  startup:
+    enabled: true
```

See initial comment. 

```suggestion
readinessProbe:
  tcpSocket:
    port: http
  periodSeconds: 15
livenessProbe:
  abc: xyz
startupProbe:
  abc: xyz
  ...
```